js/algoliasearch/internals/frontend/common.js

```diff
@@ -143,17 +142,33 @@ document.addEventListener("DOMContentLoaded", function (e) {
 				hit.price = hit.price[0];
 			}
 
-			if (hit['price'] !== undefined && price_key !== '.' + algoliaConfig.currencyCode + '.default' && hit['price'][algoliaConfig.currencyCode][price_key.substr(1) + '_formated'] !== hit['price'][algoliaConfig.currencyCode]['default_formated']) {
```

Comment:
So I could't quite get my head around what was going on with this if statement but all references from here onwards to price_key were broken because they expected price key to be .default or .group_0, when price key is .currency.key, so I removed the if statement